ocaml/tests/test_observer.ml

```diff
@@ -405,6 +404,13 @@ let test_hashtbl_leaks () =
   let test_trace_log_dir = trace_log_dir ~test_name:"test_hashtbl_leaks" () in
   let __context = Test_common.make_test_database () in
   let self = test_create ~__context ~enabled:true () in
+  let filter_export_spans span =
+    match Span.get_name span with
+    | "Tracing.flush_spans" | "Tracing.File.export" | "Tracing.Http.export" ->
```

Comment:
These are very specific names. Can we be sure they arrive exactly like this? Otherwise it would be good to downcase them before the match.

Reply:
I've downcased them before matching.